test/Tests/src/System/File_Spec.enso

```diff
@@ -675,21 +675,23 @@ spec =
             f1 = transient / "./test1.txt"
             f2 = File.new ("parentlesstest" + Date_Time.now.to_unix_epoch_milliseconds.to_text + ".txt")
 
-            txt = "MY CONTENT"
-            txt.write f1
-            txt.write f2
+            cleanup =
+                f2.delete
 
-            f1.read_text . should_equal txt
-            f2.read_text . should_equal txt
+            Panic.with_finalizer cleanup <|
```

Comment:
IMHO for such simple case, I'd avoid the complexity of the 0-argument function block `cleanup` and just put the operation verbatim:
```suggestion
            Panic.with_finalizer f2.delete <|
```

But yours is OK too.